container/state/state.go

```diff
@@ -1,4 +1,4 @@
-package container
```

Comment:
I'm doing this for better code readability.
In our existing code, we usually call state checking function like this:
```
var container *container.Container
if container.IsRunningLocking() {...}
```
Before I move the state.go to a separate directory, on a first sight, `container.IsRunningLocking()` can be a global function under package container, or a function belonging to `container.Container`, that confused me from time to time.